doc/server-secrets.md

```diff
@@ -147,6 +147,15 @@ These settings are used by shields.io for GitHub OAuth app authorization
 but will not be necessary for most self-hosted installations. See
 [production-hosting.md](./production-hosting.md).
 
+### GitLab
+
+- `GITLAB_ORIGINS` (yml: `public.services.drone.authorizedOrigins`)
+- `GITLAB_TOKEN` (yml: `private.drone_token`)
```

Comment:
Minor copy & paste errors in the docs here (drone)

Reply:
Fixed in ee81adcc8bce2f271d444ed6f144c44f9dff6755